README.md

```diff
@@ -1,12 +1,75 @@
-# DuckDuckHack Spice Instant Answers [![Build Status](https://travis-ci.org/duckduckgo/zeroclickinfo-spice.png?branch=bttf)](https://travis-ci.org/duckduckgo/zeroclickinfo-spice)
+# Welcome to DuckDuckHack's Spice Instant Answers Repository
 
-Join us on Slack! [Request invite](mailto:QuackSlack@duckduckgo.com?subject=AddMe)
+![](http://docs.duckduckhack.com/assets/hack search engine.png)
 
-This repository contains all the Spice instant answers. If you are developing a Spice instant answer you will need to fork this repository.
+**Instant Answers are an opportunity for anyone to create their own search results.**
 
-**If you would like to contribute to DuckDuckHack, please start by reading the [DuckDuckHack Documentation](https://duck.co/duckduckhack/ddh-intro).**
+Instant Answers appear on DuckDuckGo.com above ads and organic links, on [millions of searches](https://duckduckgo.com/traffic.html). Instant Answers are created by an open source community of developers [around the world](http://duckduckgo.meetup.com/) like you. Welcome!
 
-------
+You can find the [full documentation here](http://docs.duckduckhack.com).
 
-### Spice Instant Answer Example
-![App Search Example](https://raw.githubusercontent.com/duckduckgo/duckduckgo-documentation/master/duckduckhack/assets/spice_readme_example.png)
+[![slack](http://docs.duckduckhack.com/assets/slack.png) Talk to us on Slack](mailto:QuackSlack@duckduckgo.com?subject=AddMe) or [email us](mailto:open@duckduckgo.com).
```

Comment:
Did you mean to keep this here? We pushed it to the bottom in Goodies. It may help to have one near the top though?